### ISSUE TITLE
Base parameters for replication of decoder

### ISSUE THREAD

**Issue by Veldrovive** [lucidrains/DALLE2-pytorch#137]
This [run](https://wandb.ai/veldrovive/dalle2_train_decoder/runs/2gs6020x) has verified distributed training for the first unet of the decoder and I would like to set out some standard parameters that can be experimented with going forward.

The above run uses embeddings from vit-l-14 and has unet parameters:
`dim=512`
`dim_mults=(1, 2, 3, 4)`
`attn_dim_head = 32`
`attn_heads = 16`
`resnet_groups = 8`
`num_resnet_blocks = 2`
`init_cross_embed_kernel_sizes = (3, 7, 15)`

I have no reference point for what could be changed to improve performance. If anyone has magic knowledge of what hyperparameter values might improve performance it would really go a long way to making the guesswork process quicker.

**Comment by lucidrains** [comment]
@Veldrovive it is safe to just go with the same hyperparameters as what Imagen has, as Imagen outperforms DALLE2 anyways

**Comment by YUHANG-Ma** [comment]
@Veldrovive Hi, could I ask how you set your lr? as a value or lr_decay?

**Comment by YUHANG-Ma** [comment]
Hi @Veldrovive , I am training the decoder part. I saw your result in https://wandb.ai/veldrovive/dalle2_train_decoder/runs/jkrtg0so?workspace=user-veldrovive. Your loss jumps down so fast to 0.15 after training on about 3w samples. However, in my training, the loss decays so slow and it stops declining at 0.38. I use the same hyperparameters as you mentioned above and I didn't change the code. Could I ask how you set your training to make your loss that low?
